backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientLinkMutationResolver.java

```diff
@@ -25,8 +24,4 @@ public String sendPatientLinkSms(UUID internalId) throws NumberParseException {
         internalId,
         "Please fill out your Covid-19 pre-test questionnaire: " + patientLinkUrl + internalId);
   }
-
-  public PatientLink refreshPatientLink(String internalId) {
-    return pls.refreshPatientLink(UUID.fromString(internalId));
```

Comment:
Is there a chance we will want to support a feature to allow providers to renew an expired link in the future?

Reply:
Maybe? But I figure it’s just as easy to send a _new_ link as it is to refresh an existing one, and there’s already an implemented UX for that flow

Reply:
Is it possible to send a new link after the initial one is created when the test order is added to the queue? I don't recall implementing that :thinking:

Reply:
Oh, good call, I guess it actually just re-sends the same link. This is easy to fix though!

Reply:
What I settled on here is always refreshing PatientLinks when the provider initiates an SMS. What do you think?

Reply:
Perfect!